examples/PasswordExpiryWatcher/README.md

```diff
@@ -0,0 +1,96 @@
+## Password Expiry Watcher
+
+This tool allows you as a system administrator to download a CSV of all of your users with each user's password expiration date. This allows monitoring of users with upcoming password expirations and for you the administrator to prevent these users from abruptly being cut out of their mission critical systems.
+
+### To Install
+
+##### I do not have the Go toolchain installed
+> Most users will want to follow these instructions
+
+1. Go to the "Releases" tab of this github project
+	- https://github.com/TheJumpCloud/jcapi/releases
+2. We have provided `.zip` files for most operating systems
+	- Note that macOS users will want to use the `darwin` binaries
+3. Download the zip file of your choice 
+	- We always recommend downloading from the latest release if possible
+	- Most Windows users will want the `386` zip
+4. Extract the files from the zip
+	- Right click on the zip file and click `Extract All` on Windows
+	- Double click on the zip file on macOS/Linux
+5. `PasswordExpiryWatcher_myos_myarch` is the relevant binary
+	- Some may find this is an unwieldy name to type every time. Feel free to rename to your liking, in fact if this is a tool you plan on running frequently we encourage this for ease of use.
+
+> If you'd like to be able to call this binary from an arbitrary directory you can move it to `/usr/local/bin` on Linux and macOS
```

Comment:
On Windows, you could add it to C:\Windows\System32\ or anywhere else listed in %PATH%. Dunno if you'd really want to suggest that to customers, but it's an option.